client/src/App.js

```diff
@@ -1,31 +1,40 @@
-import React, { useReducer } from "react";
+import React, { useReducer, useEffect } from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import reducer from "./reducers";
 import initState from "./initState";
-
 import { userContext } from "./userContext";
 import { theme } from "./themes/theme";
 import { ProtectedRoute } from "./components/ProtectedRoute";
 import Login from "./pages/Login";
 import SignUp from "./pages/SignUp";
-import OnBoard from './pages/OnBoard';
+import OnBoard from "./pages/OnBoard";
+import { userGet } from "./services/userService";
 
 import "./App.css";
 
 function App() {
   const [state, dispatch] = useReducer(reducer, {}, initState);
+  useEffect(() => {
+    userGet()(dispatch);
+  }, [dispatch]);
+  console.log(state);
 
   return (
-    <userContext.Provider value={state.user}>
+    <userContext.Provider value={(state, dispatch)}>
```

Comment:
This change removes the value of the context provider. Putting state in here triggers a rerender of the entire component tree on state change.